samples/scan_rt_bm/Kconfig

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+menu "nRF70 BM scan and radio test combo sample configuration"
+
+menu "Zephyr Kernel"
+source "Kconfig.zephyr"
+endmenu
+
+module = NR70_SCAN_RT_BM_SAMPLE
```

Comment:
Any reason why the nomenclature does not include _COMBO_ or _COMBINED_ and just lists the two modes, _SCAN_RT?

Reply:
No specific reason :). Felt that having both scan and RT in the name implied combined.